metno/__init__.py

```diff
@@ -138,8 +138,25 @@ def get_weather(self, time, max_hour=6):
         res['humidity'] = get_data('humidity', ordered_entries)
         res['wind_speed'] = get_data('windSpeed', ordered_entries)
         res['wind_bearing'] = get_data('windDirection', ordered_entries)
+        minmax = get_min_max(ordered_entries)
+        res['templow'] = minmax['min']
```

Comment:
Why do you have to search for min temperature?
Is that not given in the data from met.no?

Reply:
I found attributes "minTemperature" and "maxTemperature" in the API response, but to me it looked like these indicate that they predict a temperature for some particular timespan that falls within this range (min-max). So as far as I can see, there's no "minTemperature" or "maxTemperature" on a per-day basis.